hackathon_site/dashboard/frontend/src/components/general/Header/Header.test.js

```diff
@@ -1,19 +1,20 @@
 import React from "react";
-import { render } from "@testing-library/react";
+
+import { render } from "testing/utils";
+
 import Header from "./Header";
-import { withStoreAndRouter } from "testing/helpers";
 
 describe("<Header />", () => {
     test("renders header with navbar", () => {
-        const { getByText, getByTestId } = render(withStoreAndRouter(<Header />));
+        const { getByText, getByTestId } = render(<Header />);
         expect(getByText(/Hackathon Name/i)).toBeInTheDocument();
         expect(getByTestId("headerLogo")).toBeInTheDocument();
         expect(getByText("Dashboard")).toBeInTheDocument(); // Checking if navbar is in header
     });
 
     test("renders header with no navbar", () => {
         const { getByText, getByTestId, queryByText } = render(
-            withStoreAndRouter(<Header showNavbar={false} />)
```

Comment:
hey, what was the rationale behind removing the "withStoreAndRouter" wrapper from all of the HTML elements? It seems like there was a new store/router function added, so why choose to remove "withStoreAndRouter" across the test files?

Reply:
Though it doesn't show up in the diff of this line, the `render` function is actually a different one now - notice at the top of the diff for pretty much all the test files, I import render from `testing/utils` now (in that file, it's defined as `customRender` and exported as `render`).

Since we are moving towards the recommended approach of testing all components with an actual redux store (as opposed to testing unconnected components like we used to), pretty much every render we do will need a store (and the router, which makes pages work). Since we'll need them everywhere, I just baked them in to the new custom `render` function.